docs/integrations/delta.md

```diff
@@ -20,7 +20,17 @@ Because lakeFS is format-agnostic, you can save data in Delta format within a la
 
 {% include toc.html %}
 
-## Beta: Viewing Delta table changes in lakeFS 
+## Beta: Viewing Delta Lake table changes in lakeFS
+
+### Installing the Delta Lake diff plugin
+
+To enable the Delta Lake diff feature, you need to install a plugin. You will find the plugin binary in the
+[release](https://github.com/treeverse/lakeFS/releases/latest) tarball (versions >= 0.100.0).
+Put the `delta_diff` binary under `~/.lakefs/plugins/diff` on the machine where lakeFS is running. 
```

Comment:
really in the user's home folder? it might be worth explaining (briefly) in the text why it's like this and not with the rest of the lakeFS binaries.